content/04.results.md

```diff
@@ -112,5 +112,32 @@ A perfectly calibrated model would follow the dashed diagonal line.
 The blue line represents the predictions before calibration and the orange line shows predictions after calibration. 
 ](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/model_calibration_experiment/model_calibration.png){#fig:discriminative_model_calibration}
 
-## Reconstructing Hetionet
-place figure of number of new edges that can be added to hetionet as well as edges we can reconstruct using this method
+
+### Baseline Comparison
+Once our discriminator model is calibrated, we compared our model's ability to predict edges in our test set to a previosuly published baseline model [@tag:cocoscore].
+We report performance for both models in terms of AUROC and AUPR (Figure {@fig:cocoscore_comparison}).
+Across edge types our model shows comparable performance against the baseline in terms of AUROC.
+Regarding AUPR, our model shows hindered performance against the baseline.
+The exception for both cases is CtD where our model performs better than the baseline.
+
+![
+Comparion between our model and CoCoScore model [@tag:cocoscore].
+We report both model's performance in terms of area under the receiver operating curve (AUROC) and area under then precision recall curve (AUPR).
+Our model achieves comparable performance against CoCoScore in terms of AUROC.
+As for AUPR CoCoScore consistently outperforms our model except for compound treats disease (CtD). 
+](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/literature_models/model_comparison.png){#fig:cocoscore_comparison}
+
+### Reconstructing Hetionet
+
+We evaluated how many edges we can recall and add to Hetionet v1 (Figure {@fig:hetionet_reconstruction}).
+In our evaluation we used edges assigned to our test set.
+Overall, we can recall a small amount of edges at high precision thresholds.
+A key example is CbG and GiG where we recalled only one exisiting edge at 100% precision.
+Despite the low recall, we are still able to add novel edges to DaG and CtD while retaining modest precision.
+
+![
+A scatter plot showing the number of edges (log scale) we can add or recall at specified precision levels. 
+The blue depicts edges exisiting in hetionet and the orange depicts how many novel edges can be added.
+](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/edge_prediction_experiment/edges_added.png){#fig:hetionet_reconstruction}
+
+
```

Comment:
To help people understand the output and its promise, would it be a good idea to include a table of top predictions?

You could image the following columns:

- edge type
- source node
- target node
- probability (generator)
- probability (discriminator)
- number of sentences
- example sentence

This table will vary depending on what exactly has been done relating to my above question:

> The discriminator provides predictions for sentences... how do you convert from sentence predictions to edge predictions?